### DISCUSSION TITLE
Resuming At Post-Setup After Install Errors

### DISCUSSION THREAD

**Discussion by adampankow** [Security-Onion-Solutions/securityonion#4173] [Unsupported Versions] [answered]
Is there a specific process for resuming the setup steps at post-setup after errors have occurred? In my case it appeared Wazuh was already listening on a port, so the process errored out, thus none of the post-setup steps completed.

**Comment by TOoSmOotH** [comment, marked as answer]
If you re-run a state.highstate and everything shows green then you are probably ok if that was the only error.
